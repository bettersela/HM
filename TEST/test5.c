//switch-case test
void main(int n)
{
	switch (n%3)
	{
	case 0: print 0;
	case 1: print 1;
	case 2: print 2;
	}
	println;
}