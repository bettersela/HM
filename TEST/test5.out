24 19 1 5 25 13 0 0 1 11 0 3 5 0 0 6 17 23 0 0 22 15 -1 13 0 0 1 11 0 3 5 0 1 6 17 41 0 1 22 15 -1 13 0 0 1 11 0 3 5 0 2 6 17 59 0 2 22 15 -1 0 10 23 15 -1 15 0 21 0